migrate/20230801_invoices.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_table(:invoice) do
```

Comment:
I think we need a creation date for invoice.
Do we use uuid as invoice number? @fdr was mentioning some relation between taxes and invoice numbers. For example in Turkey, you need to increment invoice number one by one. If you miss a number, you have to prove to government you don't do illegal things.